kornia/feature/laf.py

```diff
@@ -6,6 +6,8 @@
 import torch
 import torch.nn.functional as F
 import warnings
+from typing import List
+
 try:
     import matplotlib.pyplot as plt
```

Comment:
what did we decide about matplotlib?

Reply:
Hmm, looks like the problem with merging "approved" PR - I have done the changes at the time,
but it was not there. Will clean-up with next PR